src/projectM-sdl/projectM_SDL_main.cpp

```diff
@@ -393,6 +394,10 @@ modKey = "CMD";
 
     app->init(win, &glCtx);
 
+	// if using built in settings then default to full screen.
+	if (configFilePath.empty())
```

Comment:
I think pmSDL mostly always uses built-in config. It doesn't install the config file by default